File: C++/SegundoParcial/Practico_4/Punto5.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){

    string palabra;

    cout << "\nIngrese una palabra: ";
    cin >> palabra;

    int i = 0;
    while (i < palabra.length())
    {
        cout << palabra[i] << " \n";
        i++;
    }

    return 0;


}

File: Python3/PrimerParcial/Practico 1/Punto5.py
"""
Dada una cadena de texto ingresada por consola, decir cuántos “espacios” contiene.
"""

texto = input("\nIngrese un cadena de texto: ")
contador = 0

for caracter in texto:
    if caracter == " ":
        contador += 1

print(f"La cadena {texto}, contiene {contador} espcaios")